controllers/install/armadaserver_controller.go

```diff
@@ -296,22 +298,118 @@ func createArmadaServerServiceAccount(as *installv1alpha1.ArmadaServer) *corev1.
 	return &sa
 }
 
-func createIngress(as *installv1alpha1.ArmadaServer) *networkingv1.Ingress {
-	return &networkingv1.Ingress{
+func createIngressGRPC(as *installv1alpha1.ArmadaServer) *networkingv1.Ingress {
+	grpcIngress := &networkingv1.Ingress{
 		//	metav1.TypeMeta `json:",inline"`
```

Comment:
If you get a chance I would remove the comments.

Reply:
Good suggestion - removed in commit 7b4ee8f